<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryViewDlg</class>
 <widget class="QDialog" name="HistoryViewDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historial de cambios</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Historial de Cambios</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Desde aquí podrá revisar los cambios efectuados por los usuarios y relacionados con el registro actual. Un cambio puede involucrar cambios en diversos registros o zonas, que se agrupan dentro de una transacción. Puede observar los datos previos o que existían antes de la modificación, y los nuevos datos introducidos. Éstos últimos se presentan en negrita.&lt;/p&gt;&lt;p&gt;Si desea ver sólo los campos cambiados, puede seleccionar el check box que se encuentra al pie del formulario.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Visualizar sólo los campos que han cambiado</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbOk">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/aplicacion/images/ok.png</normaloff>:/aplicacion/images/ok.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
